controller/src/main/java/io/pravega/controller/store/stream/ZKScope.java

```diff
@@ -298,13 +298,13 @@ protected void declareVersions() {
             private void read00(RevisionDataInput revisionDataInput, Token.TokenBuilder builder) throws IOException {
                 builder.msb(revisionDataInput.readCompactInt())
                        .middle(revisionDataInput.readCompactInt())
-                       .lsb(revisionDataInput.readCompactInt());
+                       .lsb(revisionDataInput.readInt());
             }
 
             private void write00(Token token, RevisionDataOutput revisionDataOutput) throws IOException {
                 revisionDataOutput.writeCompactInt(token.msb);
                 revisionDataOutput.writeCompactInt(token.middle);
-                revisionDataOutput.writeCompactInt(token.lsb);
+                revisionDataOutput.writeInt(token.lsb);
```

Comment:
I don't understand this change either.

Reply:
Same as before.. we write and read int instead of compact int